Document/0x05b-Basic-Security_Testing.md

```diff
@@ -302,15 +337,112 @@ uid=0(root) gid=0(root) groups=0(root),1004(input),1007(log),1011(adb),1015(sdca
 
 Rooting an emulator is therefore unnecessary; root access can be established with `adb`.
 
-##### Restrictions When Testing on an Emulator
+#### Host-Device Data Transfer
+TODO `adb push pull`, scp. https://developer.android.com/studio/command-line/adb#copyfiles
```

Comment:
what about tools like House? Should also explore that and at least reference it here

Reply:
I'm not sure if the comment is related to this like about Data transfer. House can be used for runtime analysis with Frida. We can add it as  TODO (issue) to take a closer look and decide if we want to add it.

Regarding Data Transfer, the basics are adb pull/push but I'm thinking about something like FTP, I'm currently working on a blog post about that so I'll be able to provide the details for the guide. It's actually very easy and you can just access the Android File system from e.g. Finder on macOS:

![android_ftp_from_android](https://user-images.githubusercontent.com/29175115/58373038-5a7b6880-7f27-11e9-8203-89488b1183ab.png)
<img width="564" alt="android_ftp_from_macos" src="https://user-images.githubusercontent.com/29175115/58373039-5a7b6880-7f27-11e9-9a34-7f67231e3f2b.png">

Reply:
Agree, adb pull/push is the easiest and main way to handle file transfers and should be mentioned first.
Regarding House, I think the last time I used it I was able to download files quite easily (hope I don't mix it up with passionfruit). So might make sense to mention this here also as another way of accessing files and we verify it. 
Another thing is objection. It's quite tedious, as you can only download files one by one, but maybe also worth double checking the latest objection release. Maybe this has also become easier now. 

Your approach with FTP is quite cool. If this is stable and working as expected we should definitely mention that in the guide.

Reply:
I'd also add objection. Also because it has the benefit of letting us access the app sandbox on non-rooted devices; And there's also the Android Studio Device File Manager we could also add as it already comes with Android Studio

<img width="620" alt="Captura de pantalla 2019-05-26 a las 13 40 44" src="https://user-images.githubusercontent.com/29175115/58381310-066e9380-7fbc-11e9-9541-0625436ed377.png">

Reply:
I'll just include all these options for the issue and then we can evaluate and discuss further over there :)